docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -80,6 +59,7 @@ A Yugabyte Cloud account is limited to a single free cluster; you can add as man
 | Fault Tolerance | None (Single node, RF-1) | Multi node RF-3 clusters with Availability zone and Node level |
 | Scaling | None | Horizontal and Vertical |
 | Backups | None | Scheduled and on-demand |
+| YugabyteDB | Latest | Stable |
```

Comment:
Maybe want to add "version" here? I dunno. The rest of the entries are super-terse, so maybe not. Maybe another good spot to link to release notes, too.
```suggestion
| YugabyteDB version | Latest | Stable |
```